admin_manual/ai/app_context_chat.rst

```diff
@@ -69,15 +83,15 @@ Context chat will automatically load user data into the Vector DB using backgrou
 
 .. code-block::
 
-   occ background-job:worker 'OCA\ContextChat\BackgroundJobs\StorageCrawlJob'
+   occ background-job:worker -t 60 'OCA\ContextChat\BackgroundJobs\StorageCrawlJob'
```

Comment:
Not sure if `-t 60` should be added here because the occ command does not run in a loop.